### PR TITLE
Change metadata function to accept argument configuring request signing.

### PR DESCRIPTION
**Pull request by e45lee**

### CODE REVIEW

File: src/saml20_clj/sp.clj [hunk 1]
```diff
@@ -42,31 +42,37 @@
...
+(defn metadata
+  ([app-name acs-uri certificate-str sign-request?]
+   (str
+     (hiccup.page/xml-declaration "UTF-8")
+     (hiccup/html
+       [:md:EntityDescriptor {:xmlns:md  "urn:oasis:names:tc:SAML:2.0:metadata",
+                              :ID  (clojure.string/replace acs-uri #"[:/]" "_") ,
+                              :entityID  app-name}
+        [:md:SPSSODescriptor
+         (if sign-request? {:AuthnRequestsSigned "true",
```
Comment (Elknar) [inline src/saml20_clj/sp.clj]: `cond->` could've helped not having to copy the 2 other lines

**Review by Elknar — APPROVED** [review]
otherwise looks fine, not merging yet – no clue what the procedure should be.

File: src/saml20_clj/sp.clj (hunk 1 above)
Comment (jpgauthier) [reply src/saml20_clj/sp.clj]: Please change this and we'll bump this library and deploy a new version

**Comment by jpgauthier** [comment]
We'll also need to make this project deploy to clojars via Travis. Let me know when you guys are ready.

**Comment by e45lee** [comment]
Alright, that should take care of the PR comments.  Is there a template travis.yml file that we use for this?

**Comment by jpgauthier** [comment]
Indeed, all our open-source projects have a tavis file https://github.com/kirasystems/views-honeysql/blob/master/.travis.yml and some configurations in the project.clj files https://github.com/kirasystems/views-honeysql/blob/master/project.clj

**Comment by e45lee** [comment]
@jpgauthier @Elknar I've added the Travis build file and the release task modifications to the PR.

File: project.clj [hunk 2]
```diff
@@ -1,4 +1,4 @@
-(defproject kirasystems/saml20-clj "0.1.10"
```
Comment (jpgauthier) [inline project.clj:1]: Can you change the version to `0.1.11-SNAPSHOT`?

Comment (e45lee) [reply project.clj:1]: Done

**Comment by e45lee** [comment]
Alright, @jpgauthier that should address the last of your PR comments.

**Pull request merged**